packages/flutter/test/material/expansion_panel_test.dart

```diff
@@ -1343,4 +1346,54 @@ void main() {
     expect(box.size.height, equals(128.0)); // _kPanelHeaderCollapsedHeight + 80.0 (double padding)
     expect(box.size.width, equals(736.0));
   });
+
+  testWidgets('ExpansionPanelList respect dividerColor', (WidgetTester tester) async {
+    const Color dividerColor = Colors.red;
+    await tester.pumpWidget(const MaterialApp(
+      home: SingleChildScrollView(
+        child: SimpleExpansionPanelListTestWidget(
+          dividerColor: dividerColor,
+        ),
+      ),
+    ));
+
+    final DecoratedBox decoratedBox = tester.widget(find.byType(DecoratedBox).last);
+    final BoxDecoration decoration = decoratedBox.decoration as BoxDecoration;
+
+    // For the last DecoratedBox, we will have a Border.top.
```

Comment:
nit: slightly more description
```suggestion
    // For the last DecoratedBox, we will have a Border.top with the provided dividerColor.
```